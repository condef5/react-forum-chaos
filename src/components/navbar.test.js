import React from "react";
import { render } from "@testing-library/react";

import Navbar from "./navbar";

test("Navbar component", () => {
  const user = { username: "lian", email: "liamrn94@mail.com" };

  const { asFragment } = render(<Navbar user={user} />);

  expect(asFragment()).toMatchSnapshot();
});
