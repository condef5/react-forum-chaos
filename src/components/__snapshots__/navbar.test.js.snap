// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component 1`] = `
<DocumentFragment>
  <nav>
    <a
      href="/"
    >
      <span>
        Logo
      </span>
    </a>
    <ul>
      <li>
        <a
          aria-current="page"
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="#"
        >
          Logout
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
